glossary.yml

```diff
@@ -10487,6 +10487,12 @@
       exhibited through perceiving, synthesizing and inference of information. Example tasks include
       [natural language processing](#nlp), computer vision, and [machine learning]
       (#machine_learning).
+  uk:
+    term: "штучний інтелект (ШІ)"
+    def: >
+      Інтелект, який демонструють машини, на відміну від людей або інших тварин. ШІ може проявлятися
+      через сприйняття, синтезування та виведення інформації. Прикладом завдань є [оброботка природньої мови](#nlp),
```

Comment:
I think it's better to say "Серед завдань можна виділити" instead of "Прикладом завдань".

Both options are good, but "Серед завдань можна виділити" sounds more formal and polished, which might be more suitable.

Reply:
також оброботка -> обробка

Reply:
Thanks for corrections. Argeed with both!